_includes/custom/accelerometer.html

```diff
@@ -14,7 +14,12 @@
 {% elsif {{page.boardname}} == "!Fly-CDYv3" {{page.boardname}} == "!Fly-Super8" %}
 Because the firmware is so flexible and we're using software SPI, most pins can be used for this.  
 {% endif %}
-These connections are on the {{page.location}} on the board.  
+These connections are on the {{page.location}} on the board.
+
+Keep in mind that many modules (especially the purple ones from ebay or aliexpress) don't have a voltage regulator, so connect them to 3.3 V and check the voltage before plugging it in.

```

Comment:
pr looks good but you can shorten your additions a bit, making it look more formal:
keep in mind ~~that many~~ **some** modules ( ~~especially~~ **E.g.** [...]), ~~so~~ connect [...]

Reply:
ok please check again :)